docs/content/querying/filters.md

```diff
@@ -58,9 +58,9 @@ The filter specified at field can be any other filter defined on this page.
 
 ### JavaScript filter
 
-The JavaScript filter matches a dimension against the specified JavaScript function predicate. The filter matches values for which the function returns true.
+The JavaScript filter matches dimensions against the specified JavaScript function predicate. The filter matches values for which the function returns true.
 
-The function takes a single argument, the dimension value, and returns either true or false.
+The function takes the same number of arguments as the dimension values, and returns either true or false.
 
 ```json
 {
```

Comment:
we should remove the old form from the documentation and only maintain it for backwards compatibility